hack/update-bazel.sh

```diff
@@ -62,11 +62,15 @@ for repo in "${staging_repos[@]}"; do
 done
 
 # Run gazelle to update Go rules in BUILD files.
+# filter out known pkg-config error (see buildozer workaround below)
+# NOTE: the `|| exit "${PIPESTATUS[0]}"` is to ignore grep errors 
+# while preserving the exit code of gazelle
 gazelle fix \
     -external=vendored \
     -mode=fix \
     -repo_root "${KUBE_ROOT}" \
-    "${KUBE_ROOT}"
+    "${KUBE_ROOT}" \
+    2>&1 | grep -Ev "vendor/github.com/seccomp/libseccomp-golang/seccomp(_internal)?.go: pkg-config not supported: #cgo pkg-config: libseccomp" || (exit "${PIPESTATUS[0]}")
```

Comment:
Open to better ideas for this. `grep` sets an exit code itself based on matching and I couldn't find an option to prevent this. AFAICT this works as intended though.

I tested adding bogus flags to `gazelle` to reliably cause an error and the script exited with an error properly with this workaround.

Reply:
original thread about possibly doing something like this, without much response ... https://github.com/kubernetes/kubernetes/pull/79287#discussion_r296493509

Reply:
Looks like this works. 👍 

I guess it would also be possible to store the output in a variable instead of sending it through `grep` via a pipe? That should solve the problem with the return value too. Not sure what the optimal solution is tho

```bash
GAZELLE_OUT=$(gazelle fix ....)

grep -Ev "pattern" <<< ${GAZELLE_OUT}
```

Reply:
Since we have errexit you would never see the output on failure if we did it this way 😬

Reply:
Ahh, yee, totally forgot that! 🙃